code/game/machinery/holosign.dm

```diff
@@ -67,6 +67,28 @@ var/list/obj/machinery/holosign/holosigns = list()
 	id_tag = "virology"
 	light_color = "#59FF79"
 
+/obj/machinery/holosign/morgue
+	name = "morgue holosign"
+	desc = "Small wall-mounted holographic projector. This one reads MORGUE."
+	on_icon = "morgue"
+	id_tag = "morgue"
+	var/should_update = FALSE
+
+/obj/machinery/holosign/morgue/process()
+	var/area/this_area = get_area(src)
+	for(var/obj/structure/morgue in morgue_list)
+		var/area/morgue_area = get_area(morgue)
+		if(morgue_area != this_area)
+			continue
+		var/morgue_has_revivable_dude = morgue.icon_state == "morgue4"
```

Comment:
The lack of parenthesis to this statement upsets me

Reply:
```suggestion
		var/morgue_has_revivable_dude = (morgue.icon_state == "morgue4")
```